common/workspace-hack/Cargo.toml

```diff
@@ -14,6 +14,7 @@ edition = "2018"
 Inflector = { version = "0.11.4", features = ["default", "heavyweight", "lazy_static", "regex"] }
 anyhow = { version = "1.0.40", features = ["backtrace", "default", "std"] }
 backtrace = { version = "0.3.56", features = ["addr2line", "default", "gimli-symbolize", "miniz_oxide", "object", "serde", "std"] }
+bitvec = { version = "0.19.5", features = ["alloc", "atomic", "default", "std"] }
```

Comment:
Are these irrelevant?

Reply:
> `cargo build` on my Intel mac went through, so as long as CI passes we are good.

Yes, it also works on my previous intel mac. The issue with rocksDB is that it assumed __linux__ when arm64 is present.

> Are these irrelevant?

I think this is generated by my IDE or `cargo x generate-workspace-hack`.

Reply:
It seems that the land-blocking script is not satisfied with Cargo.lock